pkg/operator/etcd/pod.go

```diff
@@ -226,6 +235,17 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 	obj.Spec.Subdomain = PeerServiceName(spec.LockserverName)
 	obj.Spec.ImagePullSecrets = spec.ImagePullSecrets
 
+	if planetscalev2.DefaultEtcdFSGroup > 0 {
+		if obj.Spec.SecurityContext == nil {
+			obj.Spec.SecurityContext = &corev1.PodSecurityContext{}
+		}
+		obj.Spec.SecurityContext.FSGroup = pointer.Int64Ptr(planetscalev2.DefaultEtcdFSGroup)
```

Comment:
Is zero not a valid security context? Why not set `obj.Spec.SecurityContext.FSGroup` to a pointer of `DefaultEtcdFSGroup` no matter what?

Reply:
Since `FSGroup` is a pointer, there's a difference between 0 and nil, and I want the default default to remain nil as it was before.

I didn't see a way to tell the flag parser to leave the value nil by default, but maybe I should avoid using 0 to mean nil since 0 is a valid value. I'll change it to treat 0 as valid and `<0` as nil. I'll do the same for `RunAsUser`.

Reply:
Ok great. That was my thought - that a user could specify 0, so it should be treated as a valid value rather than assuming that means unset.